indigo_app/forms.py

```diff
@@ -137,6 +142,8 @@ def filter_queryset(self, queryset, frbr_uri=None):
 
         if self.cleaned_data.get('state'):
             queryset = queryset.filter(state__in=self.cleaned_data['state'])
```

Comment:
I don't think this does the right thing when 'assigned' is chosen. We should probably have options for 'Assigned' and 'Unassigned', and then handle them specially.